api/hearings/data/hearingLists.mock.data.ts

```diff
@@ -316,11 +316,37 @@ const HEARING_DAY_SCHEDULE_13: HearingDayScheduleModel = {
   ],
 };
 
+const HEARING_DAY_SCHEDULE_14: HearingDayScheduleModel = {
+  hearingStartDateTime: '2022-02-28T09:00:00.000+0000',
+  hearingEndDateTime: '2021-02-28T16:00:00.000+0000',
+  listAssistSessionID: '0d22d836-b25a-11eb-a18c-f2d58a9b7b18',
+  hearingVenueId: '815833',
+  hearingRoomId: 'room 5',
+  hearingJudgeId: 'hearingJudgeId1',
+  panelMemberIds: ['hearingJudgeId1'],
+  attendees: [
+    {
+      partyID: 'P1',
+      partyName: 'Jane and Smith',
+      partyType: PartyType.IND,
+      partyRole: 'claimant',
+      partyChannel: 'inPerson',
```

Comment:
`partyChannel` has been removed from the `PartiesModel` refer to https://github.com/hmcts/rpx-xui-webapp/pull/1919/files

Reply:
Sure @kuda1992 👍